### PR TITLE
Added "curated dashboards" information and broke down, rearranged topics.

### PR DESCRIPTION
**Pull request by achatterjee-grafana**
Some of the changes include:
**Broke down one topic "Using Google Cloud Monitoring in Grafana" into 2.
**New topic is "Preconfigured Cloud Monitoring dashboards".
**Resized the images.
**Removed Add data source section.
**Moved section "Deep linking from Grafana panels to the Metrics Explorer in Google Cloud Console" to Metric queries.

### CODE REVIEW

File: docs/sources/datasources/google-cloud-monitoring/preconfig-cloud-monitoring-dashboards.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+# Preconfigured dashboards
+
+Google Cloud Monitoring data source ships with [out-of-the-box](#out-of-the-box-dashboards) and [curated](#curated-dashboards) dashboards. See also [Using Google Cloud Monitoring in Grafana]({{< relref "./_index.md" >}}) for detailed instructions on how to add and configure the data source.
+
+## Out-of-the-box dashboards
+
+Grafana v7.3 and later versions supports five of the most popular out-of-the-box GCP services. They are:
+
+- BigQuery
```
Comment (sunker) [inline docs/sources/datasources/google-cloud-monitoring/preconfig-cloud-monitoring-dashboards.md:17]: Sorry Jita, I didn't know we already have the ## Out-of-the-box dashboards section. Out-of-the-box dashboards are the same thing as curated dashboards, so we need to remove the whole ##Out-of-the-box section.

File: docs/sources/datasources/google-cloud-monitoring/preconfig-cloud-monitoring-dashboards.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+- Google Compute Engine `GCE`
+- Google Kubernetes Engine `GKE`
+
+To import the out-of-the-box dashboards:
+
+1. Go to the configuration page of a Cloud monitoring data source and click on the `Dashboards` tab.
+
+2. Click `Import` for the dashboard you would like to use.
+
+The data source of the newly created dashboard panels will be the one selected above. The dashboards have a template variable that is populated with the projects accessible by the configured service account every time the dashboard is loaded. After the dashboard is loaded, you can select the project you prefer from the drop-down list.
```
Comment (sunker) [inline docs/sources/datasources/google-cloud-monitoring/preconfig-cloud-monitoring-dashboards.md:29]: This paragraph could be moved to the ## Curated dashboards section (after Click `Import` for the dashboard you would like to use)

File: docs/sources/datasources/google-cloud-monitoring/preconfig-cloud-monitoring-dashboards.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+In case you want to customize a dashboard, we recommend that you save it under a different name. Otherwise the dashboard will be overwritten when a new version of the dashboard is released.
+
+{{< docs-imagebox img="/img/docs/v73/cloud-monitoring-dashboard-import.png" max-width= "700px" caption="Cloud Monitoring dashboard import" >}}
+
+## Curated dashboards
+
+Grafana v7.4 and later versions support some of the most popular GCP services. These curated dashboards are based on similar dashboards in the GCP dashboard samples repository.
```
Comment (sunker) [inline docs/sources/datasources/google-cloud-monitoring/preconfig-cloud-monitoring-dashboards.md:37]: I'm not sure about the word "support" here. It's not like this wasn't supported before, it's just that now they are pre-configured when you install the datasource. 

```suggestion
The Cloud Monitoring data source ships with pre-configured dashboards for some of the most popular GCP services. These curated dashboards are based on similar dashboards in the GCP dashboard samples repository.
```

**Review by sunker — CHANGES_REQUESTED** [review]

File: docs/sources/datasources/google-cloud-monitoring/preconfig-cloud-monitoring-dashboards.md (hunk 1 above)
Comment (achatterjee-grafana) [reply docs/sources/datasources/google-cloud-monitoring/preconfig-cloud-monitoring-dashboards.md:17]: @sunker, thank you for the review. The original topic was so long, it was easy to overlook existing content. Let me know if you have any other comments/suggestions.

**Comment by achatterjee-grafana** [comment]
A question to reviewers here: Do we need the section on "Create a GCP Service Account for a Project". Can we direct users to GCP documentation, maybe a link to "https://cloud.google.com/iam/docs/creating-managing-service-account-keys"? 
This will free up valuable space?

**Comment by oddlittlebird** [comment]
Wherever possible, we should link to the official data source documentation for useful information like creating accounts on that service.

**Review by sunker — APPROVED** [review]
Great work @achatterjee-grafana!

File: docs/sources/datasources/google-cloud-monitoring/_index.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 +++
-title = "Cloud Monitoring"
+title = "Google Cloud Monitoring"
 description = "Guide for using Google Cloud Monitoring in Grafana"
 keywords = ["grafana", "stackdriver", "google", "guide", "cloud", "monitoring"]
 aliases = ["/docs/grafana/latest/features/datasources/stackdriver", "/docs/grafana/latest/features/datasources/cloudmonitoring/"]
```
Comment (oddlittlebird) [inline docs/sources/datasources/google-cloud-monitoring/_index.md:5]: @achatterjee-grafana - Please add the old path to the aliases list.

File: docs/sources/datasources/google-cloud-monitoring/_index.md [hunk 5]
```diff
@@ -8,27 +8,19 @@ weight = 200
 
 # Using Google Cloud Monitoring in Grafana
```
Comment (oddlittlebird) [inline docs/sources/datasources/google-cloud-monitoring/_index.md:9]: Is now a good time to update this title to the new format per your template, or do you want to wait until we do an actual rewrite of the whole thing?

**Review by oddlittlebird — COMMENTED** [review]
This change will break the link to documentation in the Google Cloud Monitoring page in Grafana.

I recommend updating that file as well either in this PR or in a separate PR immediately following this one. Hopefully the aliases will take care of that, but as we have seen, Google does not crawl links immediately.

**Comment by oddlittlebird** [comment]
Here is the file that needs the link updated: https://github.com/grafana/grafana/blob/master/public/app/plugins/datasource/cloud-monitoring/partials/config.html

Also could use some minor grammar edits, like the missing period and changing "data source" to two words.

File: docs/sources/datasources/google-cloud-monitoring/_index.md (hunk 5 above)
Comment (achatterjee-grafana) [reply docs/sources/datasources/google-cloud-monitoring/_index.md:9]: I am hoping the developer's/SME's will work on the actual re-write and give us some feedback on how the template. I will create a separate PR to change the titles of all the data source topics.

File: docs/sources/datasources/google-cloud-monitoring/_index.md (hunk 4 above)
Comment (achatterjee-grafana) [reply docs/sources/datasources/google-cloud-monitoring/_index.md:5]: Added.

File: docs/sources/datasources/google-cloud-monitoring/preconfig-cloud-monitoring-dashboards.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
++++
+title = "Preconfigred dashboards"
```
Comment (oddlittlebird) [inline docs/sources/datasources/google-cloud-monitoring/preconfig-cloud-monitoring-dashboards.md:2]:
```suggestion
title = "Preconfigured dashboards"
```

Comment (achatterjee-grafana) [reply docs/sources/datasources/google-cloud-monitoring/preconfig-cloud-monitoring-dashboards.md:2]: Done

**Comment by achatterjee-grafana** [comment]
Note: I am keeping the section "Create a GCP Service Account for a Project" since it is referenced from this file: https://github.com/grafana/grafana/blob/master/public/app/plugins/datasource/cloud-monitoring/partials/config.html

File: docs/sources/datasources/google-cloud-monitoring/_index.md [hunk 7]
```diff
@@ -38,7 +30,7 @@ There are two ways to authenticate the Google Cloud Monitoring plugin - either b
 
 To authenticate with the Google Cloud Monitoring API, you need to create a Google Cloud Platform (GCP) Service Account for the Project you want to show data for. A Grafana data source integrates with one GCP Project. If you want to visualize data from multiple GCP Projects then you need to create one data source per GCP Project.
 
-#### Enable APIs
+##### Enable APIs
```
Comment (oddlittlebird) [inline docs/sources/datasources/google-cloud-monitoring/_index.md:33]: Why did we jump from h4 to h5 here? Looks like the previous heading is h3, so this should be h4.

File: docs/sources/datasources/google-cloud-monitoring/_index.md [hunk 8]
```diff
@@ -47,31 +39,31 @@ The following APIs need to be enabled first:
 
 Click on the links above and click the `Enable` button:
 
-{{< docs-imagebox img="/img/docs/v71/cloudmonitoring_enable_api.png" class="docs-image--no-shadow" caption="Enable GCP APIs" >}}
+{{< docs-imagebox img="/img/docs/v71/cloudmonitoring_enable_api.png" max-width="450px" class="docs-image--no-shadow" caption="Enable GCP APIs" >}}
 
-#### Create a GCP Service Account for a Project
+##### Create a GCP Service Account for a Project
```
Comment (oddlittlebird) [inline docs/sources/datasources/google-cloud-monitoring/_index.md:44]: See previous comment.

File: docs/sources/datasources/google-cloud-monitoring/_index.md [hunk 9]
```diff
@@ -181,9 +173,18 @@ Example Alias By: `{{resource.type}} - {{metric.type}}`
 
 Example Result: `gce_instance - compute.googleapis.com/instance/cpu/usage_time`
 
+#### Deep linking from Grafana panels to the Metrics Explorer in Google Cloud Console
+
+>**Note:** Available in Grafana v7.1 and later versions.
```
Comment (oddlittlebird) [inline docs/sources/datasources/google-cloud-monitoring/_index.md:178]:
```suggestion
> **Note:** Available in Grafana v7.1 and later versions.
```

File: docs/sources/datasources/google-cloud-monitoring/_index.md [hunk 10]
```diff
@@ -181,9 +173,18 @@ Example Alias By: `{{resource.type}} - {{metric.type}}`
...
+
+{{< docs-imagebox img="/img/docs/v71/cloudmonitoring_deep_linking.png" max-width="500px" class="docs-image--right" caption="Google Cloud Monitoring deep linking" >}}
+
+Click on a time series in the panel to see a context menu with a link to View in Metrics Explorer in Google Cloud Console. Clicking that link opens the Metrics Explorer in the Google Cloud Console and runs the query from the Grafana panel there.
+The link navigates the user first to the Google Account Chooser and after successfully selecting an account, the user is redirected to the Metrics Explorer. The provided link is valid for any account, but it only displays the query if your account has access to the GCP project specified in the query.
+
 ### SLO (Service Level Objective) queries
 
-> Only available in Grafana v7.0+
+>**Note** Available in Grafana v7.0 and later versions.
```
Comment (oddlittlebird) [inline docs/sources/datasources/google-cloud-monitoring/_index.md:187]:
```suggestion
> **Note** Available in Grafana v7.0 and later versions.
```

File: docs/sources/datasources/google-cloud-monitoring/_index.md [hunk 11]
```diff
@@ -304,7 +305,7 @@ Example Result: `monitoring.googleapis.com/uptime_check/http_status has this val
 
 ## Configure the data source with provisioning
 
-It's now possible to configure data sources using config files with Grafana's provisioning system. You can read more about how it works and all the settings you can set for data sources on the [provisioning docs page]({{< relref "../administration/provisioning/#datasources" >}})
+It's now possible to configure data sources using config files with Grafana's provisioning system. You can read more about how it works and all the settings you can set for data sources on the [provisioning docs page]({{< relref "../../administration/provisioning/#datasources" >}})
```
Comment (oddlittlebird) [inline docs/sources/datasources/google-cloud-monitoring/_index.md:308]:
```suggestion
You can configure data sources using config files with Grafana's provisioning system. Read more about how it works and all the settings you can set for data sources on the [provisioning docs page]({{< relref "../../administration/provisioning/#datasources" >}})
```

File: docs/sources/datasources/google-cloud-monitoring/preconfig-cloud-monitoring-dashboards.md [hunk 12]
```diff
@@ -0,0 +1,26 @@
...
+Google Cloud Monitoring data source ships with pre-configured dashboards for some of the most popular GCP services. These curated dashboards are based on similar dashboards in the GCP dashboard samples repository. See also, [Using Google Cloud Monitoring in Grafana]({{< relref "./_index.md" >}}) for detailed instructions on how to add and configure the Google Cloud Monitoring data source.
+## Curated dashboards
+
+The curated dashboards are based on similar dashboards in the GCP dashboard samples repository.
+
+To import the curated dashboards:
+
+1. Go to the configuration page of your Cloud Monitoring data source and click on the `Dashboards` tab.
+
+2. Click `Import` for the dashboard you would like to use.
```
Comment (oddlittlebird) [inline docs/sources/datasources/google-cloud-monitoring/preconfig-cloud-monitoring-dashboards.md:20]:
```suggestion
1. Click **Import** for the dashboard you would like to use.
```

File: docs/sources/datasources/google-cloud-monitoring/preconfig-cloud-monitoring-dashboards.md [hunk 13]
```diff
@@ -0,0 +1,26 @@
...
+# Preconfigured Cloud Monitoring dashboards
+
+Google Cloud Monitoring data source ships with pre-configured dashboards for some of the most popular GCP services. These curated dashboards are based on similar dashboards in the GCP dashboard samples repository. See also, [Using Google Cloud Monitoring in Grafana]({{< relref "./_index.md" >}}) for detailed instructions on how to add and configure the Google Cloud Monitoring data source.
+## Curated dashboards
+
+The curated dashboards are based on similar dashboards in the GCP dashboard samples repository.
+
+To import the curated dashboards:
+
+1. Go to the configuration page of your Cloud Monitoring data source and click on the `Dashboards` tab.
```
Comment (oddlittlebird) [inline docs/sources/datasources/google-cloud-monitoring/preconfig-cloud-monitoring-dashboards.md:18]:
```suggestion
1. On the configuration page of your Cloud Monitoring data source, click the **Dashboards** tab.
```

File: docs/sources/whatsnew/whats-new-in-v5-3.md [hunk 14]
```diff
@@ -37,7 +37,7 @@ The Grafana Stackdriver plugin comes with support for automatic unit detection.
 The data source is still in the `beta` phase, meaning it's currently in active development and is still missing one important feature - templating queries.
 Please try it out, but be aware of that it might be subject to changes and possible bugs. We would love to hear your feedback.
 
-Please read [Using Google Stackdriver in Grafana]({{< relref "../datasources/cloudmonitoring/" >}}) for more detailed information on how to get started and use it.
+Please read [Using Google Stackdriver in Grafana]({{< relref "../datasources/google-cloud-monitoring/_index.md" >}}) for more detailed information on how to get started and use it.
```
Comment (oddlittlebird) [inline docs/sources/whatsnew/whats-new-in-v5-3.md:40]:
```suggestion
Refer to [Using Google Stackdriver in Grafana]({{< relref "../datasources/google-cloud-monitoring/_index.md" >}}) for more detailed information on how to get started and use it.
```

**Review by oddlittlebird — APPROVED** [review]
A few minor edits, then good to go.

File: docs/sources/datasources/google-cloud-monitoring/_index.md (hunk 7 above)
Comment (achatterjee-grafana) [reply docs/sources/datasources/google-cloud-monitoring/_index.md:33]: True - fixed.

**Pull request merged**